### DISCUSSION TITLE
Hola, estoy buscando un programador para desarrollar bot para toma de citas de asilo madridREADME.md

### DISCUSSION THREAD

**Discussion by Daniela022812** [community/community#39787] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hola, estoy buscando un programador para desarrollar bot para toma de citas de asilo madrid

**Comment by david-engelmann** [comment]
Fascinating request, this might not be the best place to find someone. I'd try a Free lancing platform

**Comment by ossanna16** [comment]
Hi @Daniela022812, we're happy you're here 😀
However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
